README.rst

```diff
@@ -133,66 +143,84 @@ For other build systems
 
 Then, from your source files, include ``xmath/xmath.h``.
 
-Unit Tests
-----------
+Unit Tests and Examples
+-----------------------
+
+This project uses CMake to build the unit test and example applications. Use the steps described above to
+configure and build the unit test and example applications. Both unit test and example projects currently target the
+xCORE.ai explorer board and x86 platforms. All unit tests are currently in the `/test/
```

Comment:
I wouldn't write xCORE.ai like that. 
1. I don't believe it's an official guideline
2. It's reeeeeeally ugly

I would go for xcore.ai

Reply:
Fixed.